cardano-api/src/Cardano/Api/IPC.hs

```diff
@@ -367,6 +373,54 @@ convLocalChainSyncClient mode =
       (fromConsensusBlock mode)
       (fromConsensusTip mode)
 
+convLocalChainSyncClientPipelined
+  :: forall mode block m a.
+     (ConsensusBlockForMode mode ~ block, Functor m)
+  => ConsensusMode mode
+  -> ChainSyncClientPipelined (BlockInMode mode) ChainPoint ChainTip m a
+  -> ChainSyncClientPipelined block (Net.Point block) (Net.Tip block) m a
+
+convLocalChainSyncClientPipelined mode =
+  mapChainSyncClientPipelined
+    (toConsensusPointInMode mode)
+    (fromConsensusPointInMode mode)
+    (fromConsensusBlock mode)
+    (fromConsensusTip mode)
+
+-- TODO move to Ouroboros.Network.Protocol.ChainSync.ClientPipelined
```

Comment:
Yeah, lets do that.

Reply:
I've created a PR: https://github.com/input-output-hk/ouroboros-network/pull/2916

Reply:
@dcoutts are we free to just bump the ouroboros-network version when the linked PR is merged, or is there some company process for this?